variables.tf

```diff
@@ -70,15 +70,15 @@ variable "bucket_arn" {
   description = "The S3 bucket ARN is required when setting use_existing_cloudtrail to true"
 }
 
-variable "bucket_enable_encryption" {
+variable "bucket_encryption_enabled" {
   type        = bool
-  default     = false
+  default     = true
   description = "Set this to `true` to enable encryption on a created S3 bucket"
 }
 
 variable "bucket_enable_logs" {
```

Comment:
Would it make sense to change this from `bucket_enable_logs` to `bucket_logs_enabled` to match the new variable formatting?